<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>facedemo1Class</class>
 <widget class="QMainWindow" name="facedemo1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1474</width>
    <height>761</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>facedemo1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="showstuinfo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>570</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>941</width>
      <height>531</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>1</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>300</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No face</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>2</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>300</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No face</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>3</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>300</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No face</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>4</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>300</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No face</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>5</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>256</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>20</width>
        <height>511</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>43</x>
        <y>300</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>No face</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="up_load_image_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>上传图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>建立数据库</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="base_image_view">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1474</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="facedemo1.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>up_load_image_button</sender>
   <signal>clicked()</signal>
   <receiver>facedemo1Class</receiver>
   <slot>UpLoadImageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>627</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>UpLoadImageButtonClicked()</slot>
 </slots>
</ui>
